content/en/blog/drivers-add-new-distro.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Add prebuilt drivers for new distro
+date: 2023-03-21
+author: Federico Di Pierro
+slug: falco-prebuilt-drivers-new-distro
+---
+
+Hi everyone!  
+Today we are going to learn how to add support for a new distro prebuilt drivers.  
+There are multiple repositories involved with it, and while most of the time it should be a pretty simple job,  
+other times it can become really cumbersome.  
+That's why we are writing this guide of course!  
```

Comment:
```suggestion
Hi everyone!  
Today we are going to learn how to add support for a new distro prebuilt drivers.
There are multiple repositories involved with it, and while most of the time it should be a pretty simple job, other times it can become really cumbersome.
That's why we are writing this guide of course!
```